packages/agents/lighthouse/src/tasks/propose/errors.ts

```diff
@@ -131,18 +131,3 @@ export class AggregateRootChecksFailed extends NxtpError {
     });
   }
 }
-
-export class SubgraphDelayed extends NxtpError {
```

Comment:
are we sure we want to remove this error completely?

what happens when carto is hours behind?